themes/src/main/themes/VAADIN/themes/valo/components/_table.scss

```diff
@@ -290,9 +291,11 @@ $v-table-background-color: null !default;
 
   .#{$primary-stylename}-table td {
     border-top: $v-table-border-width solid $border-color;
+    box-sizing: border-box;
   }
 
   .#{$primary-stylename}-table tr:first-child > td {
+    height: $v-table-row-height;
```

Comment:
This broke all my custom tables (I need to specify a tr:first-child too).
But also broke valo compact style.